pytest_reportportal/service.py

```diff
@@ -189,6 +197,8 @@ def start_launch(self,
         log.debug('ReportPortal - Start launch: request_body=%s', sl_pt)
         item_id = self.rp.start_launch(**sl_pt)
         log.debug('ReportPortal - Launch started: id=%s', item_id)
+        if not self._skip_analytics:
+            send_event(self.agent_name, self.agent_version)
```

Comment:
@rambangaru - this google analytics is new. Do we want to use this or silence it in our Owl implementation?

Since we are using this internally I'm leaning towards silencing it but I'm happy with either.

Reply:
there are many libraries that we use are using the some kind of analytics so It is not harmful so Im ok to leave it for now